cherrypy/lib/httputil.py

```diff
@@ -66,6 +68,31 @@ def protocol_from_http(protocol_str):
     return int(protocol_str[5]), int(protocol_str[7])
 
 
+def passes_if_range_check(request):
+    """Return true if the request does not have an If-Range header, or it passes the conditions contained in the header.
```

Comment:
According to PEP 257, it should explain what the function does, not how it does that. Leave out implementation details.

https://www.python.org/dev/peps/pep-0257/